src/helpers/functions.ts

```diff
@@ -106,6 +106,20 @@ export const shouldExport = (
   return false;
 };
 
+/**
+ * Checks if the given `posixPath` is not excluded by any configuration
+ *
+ * @param posixPath The path to check
+ * @returns If the given `posixPath` should be added to the list of
+ * exports
+ */
+export const shouldExportDir = (
+  posixPath: PosixPath
+): boolean => {
+  const globs: Array<string> = getConfig(CONFIGURATIONS.values.EXCLUDE_DIRS);
```

Comment:
I have been using `type[]` notation over the `Array<type>` notation. This is just a personal preference, but I will change it 😄

Reply:
My apologies. Up there it is using `Array<type>` so I was mistaken.